client/i18n/locales/english/translations.json

```diff
@@ -72,7 +72,8 @@
     "click-start-course": "Start the course",
     "click-start-project": "Start the project",
     "change-language": "Change Language",
-    "cancel-change": "Cancel Change"
+    "cancel-change": "Cancel Change",
+    "resume-project": "Resume project"
```

Comment:
There should be a "Start Project" button for projects that have not been started yet.

Reply:
Just thinking out loud here. Since we don't display any button when the project is completed, perhaps we shouldn't display any button if the project hasn't been started? A start project button is just taking you to step 1, which is just and extra tab stop to get to step 1 manually.

Reply:
I could go either way on it. I kind of like the "start project" button - it's an obvious thing to click. Otherwise, there's just a list of numbers - pretty safe to assume they will understand that 1 is what to click to start, but it will be even more obvious with the button. I don't mind either way though.

Reply:
The people have spoken!

Reply:
This has been added